R/trace_tests.R

```diff
@@ -68,23 +69,23 @@
 #' # f(!x)
 #' #
 #' # $tests
-#' #      test depth i
-#' # [1,]    1     2 4
+#' #      test call depth i
+#' # [1,]    1    1     2 4
 #'
 #' # reconstruct the code path of a test by ordering test traces by [,"i"]
 #' lapply(cov, `[[`, "tests")
 #' # $`source.Ref2326138c55:4:6:4:10:6:10:4:4`
-#' #      test depth i
-#' # [1,]    1     1 2
+#' #      test call depth i
+#' # [1,]    1    1     1 2
 #' #
```

Comment:
A change to the structure of the `<coverage>[[<key>]]$tests` elements.

Adds a `call` column, which is the number of times the test expression was evaluated before hitting this trace. It's not used within `covr`, but since the same test now represents multiple calls it is useful for distinguishing test evaluations useful for downstream tooling.